lib/lhm/invoker.rb

```diff
@@ -52,6 +52,7 @@ def run(options = {})
       entangler = Entangler.new(migration, @connection, options)
 
       entangler.run do
+        options[:verifier] = Proc.new { triggers_still_exist?(entangler) }
```

Comment:
This is definitely the worst thing -- hardcoding something into the `options` hash. I could add an extra parameter to `Chunker#initialize` to make this more explicit? The reason I want to create the verifier here is because this part of the code has easy access to all of the variables that we will need to wrap into a closure with the Proc.

Reply:
The options that exist right now are pretty much a lie. You can't invoke LHM with all the options, so this is what happens. Shrug.

Reply:
How confident are you in the implementation of `triggers_still_exist?` - it doesn't actually assert that _all_ triggers exist...just that _some_ triggers exist.

Reply:
Thanks for the review!

I'm pretty confident, it checks that the triggers that are found match `Entangler#expected-triggers` ( https://github.com/Shopify/lhm/blob/master/lib/lhm/invoker.rb#L67 and https://github.com/Shopify/lhm/blob/master/lib/lhm/entangler.rb#L77 )

And, anecdotally the current implementation of `triggers_still_exist?` has successfully avoided corruption for Shopify more than once since going in 😬